asdf_astropy/tags/transform/basic.py

```diff
@@ -0,0 +1,216 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from asdf.versioning import AsdfVersion
+import asdf
+
+from astropy.modeling import models
+from astropy.modeling.core import CompoundModel, Model
+
+from . import _parameter_to_value
+
+
+__all__ = ['TransformType', 'IdentityType', 'ConstantType']
+
+
+class TransformType(asdf.AsdfConverter):
+    tags = {"http://asdf-format.org/schemas/transform/transforms-1.0.0"}
```

Comment:
I would just leave out `tags` and `types` here, since we don't want to actually write a generic transform to a file.